src/bundle/Resources/views/fieldtypes/edit/ezdate.html.twig

```diff
@@ -7,6 +7,6 @@
             </svg>
         </button>
     </div>
-    {% set attr = attr|merge({'class': 'ez-visually-hidden ez-data-source__input'}) %}
+    {% set attr = attr|merge({'class': 'ez-data-source__input', 'hidden': null}) %}
```

Comment:
I set it to `null` in order to have `hidden` instead of `hidden="hidden"`, which is more correct? I, personally, prefer 1st option, but now I checked the codebase and most popular is `'hidden': 'hidden'`, so I switched to this notation. :)

Reply:
@tischsoic all of them: `hidden="true"`, `hidden="hidden"` and `hidden` are technically correct.